src/index.ts

```diff
@@ -278,6 +303,15 @@ export async function install(
   };
   const packageBundle = await rollup.rollup(inputOptions);
   await packageBundle.write(outputOptions);
+  if (isExplicit) {
+    styles.forEach(style => {
+      fs.copyFile(style, `${destLoc}/${path.basename(style)}`, (err) => {
```

Comment:
Replace with fs.copyFileSync for now, since we don't want @pika/web to return with a success message until  files are written.